### PR TITLE
module submission: 2024-Jarigsma-001

### PR DESCRIPTION
**Pull request by AmberHam**

### CODE REVIEW

File: 2024-Jarigsma-001/NASSA.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+id: 2024-Jarigsma-001
+nassaVersion: 0.5.0
```
Comment (Andros-Spica) [inline 2024-Jarigsma-001/NASSA.yml:2]:
```suggestion
nassaVersion: 0.5.1
```

File: 2024-Jarigsma-001/NASSA.yml
Comment (Andros-Spica) [inline 2024-Jarigsma-001/NASSA.yml:1]: Although it is not compulsory, consider adding the fields inputs and outputs, with minimal details. 
Here is my suggestion:
```
inputs:
  - name: initial-number-nodes
    type: integer
    unit: nodes
    description: the number of unconnected nodes existing before preferential attachment is applied.
outputs:
  - name: network
    type: object (agent and link sets)
    description: nodes (turtles, breed nodes) and edges (links) between nodes, formed with iterations of preferential attachment.
```

**Review by Andros-Spica — COMMENTED** [review]
Hi Amber,

I took the liberty of correcting the issues in NASSA.yml that were causing problems with the automatic checks. Have a look at my commits.

As the actual review, I made a few suggestions as comments and again took the liberty of directly committing edits on the .nlogo file, just to make it more re-usable. I did it because the changes were conceptually simple, but a bit hard to add only as suggestions and comments. Please let me know if you don't agree with the changes and we discard them.
I have also added Clemens Schmid as reviewer (hopefully he agrees with my suggestions!).

Anyway, if you agree with the changes I already commited, you can focus on the comments and suggestions (all are not critical issues).

Best,
Andreas

**Comment by AmberHam** [comment]
Hi Andros, 

I have taken a look at the changes you committed directly to the .nlogo file and I do agree that they make the code more reusable. I have also made a small change to that file but only in one of the comments by the nodes-own declaration. Other than that, I have updated the readme file according to your suggestions. I hope I have managed to address all the comments and suggestions but if I have missed anything then please let me know. Otherwise, if there are further changes that should be made, let me know also.

Best regards,
Amber

**Pull request merged**